celements-model/src/main/java/com/celements/query/CelMySQLDialect.java

```diff
@@ -0,0 +1,15 @@
+package com.celements.query;
+
+import java.sql.Types;
+
+import org.hibernate.Hibernate;
+import org.hibernate.dialect.MySQLDialect;
+
+public class CelMySQLDialect extends MySQLDialect {
```

Comment:
see [this comment](https://synjira.atlassian.net/browse/CELDEV-657?focusedCommentId=32770&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-32770) for more information